pkg/manifest/manifest.go

```diff
@@ -151,17 +156,78 @@ func (m *Manifest) Include(excludeIdentifier *string, includeTechPreview *bool,
 	return nil
 }
 
+// ManifestEnabled returns whether the manifest is enabled given the defined cluster capabilities.
+func (m *Manifest) ManifestEnabled(capabilities configv1.ClusterVersionCapabilitiesStatus) bool {
+	annotations := m.Obj.GetAnnotations()
+	if annotations == nil {
+		return true
+	}
+	enabled := true
+	caps := getManifestCapabilities(annotations)
+	for _, c := range caps {
+		_, isEnabled := getCapabilityProperties(c, capabilities)
+		if !isEnabled {
+			enabled = false
+			break
+		}
+	}
+	return enabled
+}
+
+// GetManifestCapabilities returns the manifest's capabilities.
+func (m *Manifest) GetManifestCapabilities() []configv1.ClusterVersionCapability {
+	annotations := m.Obj.GetAnnotations()
+	if annotations == nil {
+		return nil
+	}
+	return getManifestCapabilities(annotations)
```

Comment:
You can write 
```
if annonations != nil {
   return getManifestCapabilities(annotations)
}

Reply:
I can do that but normally lint would bitch about not having a return at the end of a function.